tendermint/tests/lite.rs

```diff
@@ -125,6 +197,57 @@ fn run_test_cases(cases: TestCases) {
     }
 }
 
+#[test]
+fn bisection_simple() {
+    let case: TestBisection =
+        serde_json::from_str(&read_json_fixture("many_header_bisection/happy_path")).unwrap();
```

Comment:
For each JSON we test, we should link to the Go generator that created it, so we know how it was generated. For now those generators are in Shivanis fork of tendermint Go, so we can reference it there for now, but we should work to get that stuff merged in.

Reply:
Also does the generator for this have enough validator set changes to require actual bisection (ie. intermediate requests)? We should make sure it does, and figure out a nice way to describe the different amounts of bisection we want to test for.

Reply:
Doesn't look like it. We're using the `generateNextBlock` to make the blocks:  https://github.com/Shivani912/tendermint/blob/shivani-refactor/types/lite-client/generator/single_step_utils.go#L191. We'll need some way to specify to `generateNextBlock` in the Go generators to update the validator set at certain heights. Perhaps the `privVals` we pass in should be a structure that returns the validator set for each height.

Reply:
All of this addressed in commit 98d168f. Thanks! :)